mdb.py

```
@@ -84,26 +485,263 @@ def get_all_dbs_and_tables(db, server):
+                try:
+                    query_db = {}
+                    db_connect(query_db, server=server, dictionary=False)
+
+                    # Query and get results
+                    query_db['cnf']['servers'][server]['cur'].execute(query)
+                    rows = query_db['cnf']['servers'][server]['cur'].fetchall()
+
+                    # Convert rows to dictionaries for comparison
+                    if rows:
+                        column_names = [desc[0] for desc in query_db['cnf']['servers'][server]['cur'].description]
+                        dict_rows = [dict(zip(column_names, row)) for row in rows]
```

Ruff (0.15.5)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)